src/components/ConfigWizard.js

```diff
@@ -17,13 +20,17 @@ export class ConfigWizard extends React.PureComponent {
     }
 
     handleIframeEvents = (e) => {
+        console.log(`${e.data.type} event received`);
```

Comment:
do we need this?

Reply:
Good question! I thought about leaving it since it could be helpful to see the log of received events for any new embedded partner.